### ISSUE TITLE
Low Contrast Text on Dark Background

### ISSUE THREAD

**Issue by UmeshpJadhav** [tannmayy14/ROTS#5]
Text color #E0E0E0 on dark translucent background fails accessibility contrast checks.

**Comment by PranayIppakayal** [comment]
please assign me the issue
